index.html

```diff
@@ -3,27 +3,151 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="styles/normalize.css">
-  <link rel="stylesheet" href="styles/style.css">
-  <title>Learning how to learn</title>
+  <link rel="stylesheet" href="vendor/normalize.css">
+  <link rel="stylesheet" href="pages/page.css">
+  <title>Learning how to learn</title> 
 </head>
 <body>
   <div class="page">
     <header class="header">
+      <div class="logo logo_place_header"></div>
       <h1 class="header__title">Learning How to Learn</h1>
       <p class="header__subtitle">The most effective, cutting-edge learning techniques that you can apply to your daily life
-        <a class="header__link" href="#" target="_blank">Learn more &rarr;</a>
+        <a class="link header__link" href="#description-section" target="_self">Learn more &rarr;</a>
       </p>
-      <img class="header__main-illustration" src="images/header-image.png" alt="Изображение шапки">
-      <div class="header__square-pic"></div>
+      <img class="header__main-illustration" src="images/header-image.png" alt="Путник">
+      <div class="header__square-pic rotation"></div>
     </header>
     <main class="content">
-      <section class="description">
+      <section id="description-section" class="description">
+        <div class="two-columns">
+          <h2 class="two-columns__brief">The main problem with education</h2>
+          <div class="two-columns__main-text">
+            <p class="two-columns__paragraph">Whether it's at school or university, the one thing no one ever teaches you is how to teach yourself. We often just end up drilling specific exercises and mindlessly memorizing information so that when exam time comes around, we pass the test. But once the test is over, that information has outlived its usefulness, and there's no guarantee that information will be retained. In the end, we're really just learning how to pass our tests.</p>
+            <p class="two-columns__paragraph"><span class="two-columns__span-accent">Here's the real question:</span> does this approach of repetition and memorization help us truly understand the subject material? The academic consensus nowadays is that it doesn't. In fact, what's more important than any individual topic is knowing how to study, but for some reason, this point is often overlooked. So, what can we do to remedy this?</p>
+            <p class="two-columns__paragraph">Well, there are various techniques and exercises that can be applied to the educational process to help make it more efficient and engaging. Keep scrolling to uncover some of the secrets of effective learning.</p>
+          </div>
+        </div>
+      </section>
+      <section class="techniques section techniques_padding_top_bottom">
```

Comment:
Setting up `techniques_padding_top_bottom` modifier seems a bit redundant in this case. Please consider removing it and moving styling to `techniques` itself

Reply:
Done🙂